fabric-gui-v0/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "fabric-gui-v0"
```

Comment:
I dont see this being set in the settings.gradle? Have you even tested this?

Reply:
Where's the settings.gradle? I just copied the format used by one of the existing modules.

Reply:
in the root of the project, its just a list of all the gradle projects, make sure it works and builds